README.md

```diff
@@ -233,6 +234,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 
 * [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
 * [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) Ultra performant API Gateway with middleware. Written in Go.
```

Comment:
Please add it to the end of the list

Reply:
Ok, will do.  I put it next to Tyk, as it's more similar to Tyk.  But will move down as requested.